dev-tools/scripts/releaseWizard.yaml

```diff
@@ -1306,6 +1306,11 @@ groups:
       Make sure you have a PGP enabled email client with your apache key installed.
       There are plugins for popular email programs, as well as browser plugins for webmail.
       See links for help on how to setup your email client for PGP.
+
+      If you prefer to sign the announcements manually rather than using a plugin,
```

Comment:
Fair enough to offer that choice, although you'll bypass the auto signature validation that email clients provide...